packages/sdk/identity/src/odis/query.test.ts

```diff
@@ -0,0 +1,38 @@
+import crypto from 'crypto'
+import { hexToBuffer, trimLeading0x } from '../../../base/lib'
+import { signWithRawKey } from './query'
+
+const rawKey = '41e8e8593108eeedcbded883b8af34d2f028710355c57f4c10a056b72486aa04'
+
+describe(signWithRawKey, () => {
+  it('Signs full message digest', async () => {
+    const msg = 'test'
```

Comment:
This message is too short to actually test the vulnerability. It needs to be more than 64 characters. The main vulnerability is due to message truncation. We should have a test that verifies that changes to the message invalidates the signature.